habitat_baselines/rl/ddppo/ddp_utils.py

```diff
@@ -182,9 +182,6 @@ def load_resume_state(filename_or_config: Union[Config, str]) -> Optional[Any]:
     if not osp.exists(filename):
         return None
 
-    print("error: attempt to load resume state from ", filename)
```

Comment:
Change in the code. This is necessary for auto restarting jobs on slurm and is important when I run jobs on skynet. I see no reason to leave this line in the code.